### ISSUE TITLE
Last version is again reported as infected with Wacatac.B!ml on Windows 10

### ISSUE THREAD

**Issue by szymbas** [meesoft/PhotoLocator#5]
Windows 10 still reports last version (2024.5.25.0) hosted by FossHub as infected with Wacatac.B!ml

**Comment by meesoft** [comment]
Yes, it is because I have not purchased a digital signature. Some virus
scanners doesn't like unsigned applications.

man. 3. jun. 2024 23.49 skrev szymbas ***@***.***>:

**Comment by meesoft** [comment]
I have published a zip version also so you don't have to run the installer. Perhaps that won't be flagged as a virus.

**Comment by andyzukunft** [comment]
I just loaded the setup `PhotoLocatorSetup2024.8.26.0.exe` from http://meesoft.com/PhotoLocator/.

Virus Total has some detection (https://www.virustotal.com/gui/file/3becce74891c9f12ea080ef620a0da4569959abc566888501384e6d8095eba47) however only 3 / 73 applications say "there is something". And by "something" it is detecting e.g. "Win/grayware_confidence_60% (W)" (example: CrowdStrike Falcon). "Grayware" is nothing which is "dangerous". Those anti virus scanner are just scare ware.

If in doubt use Virus Total for a quick check.
